torch/export/exported_program.py

```diff
@@ -355,7 +359,8 @@ def _decompose_and_get_gm_with_new_signature_constants(
                 fake_args.append(node.meta["val"])
 
         fake_args_unwrapped = pytree.tree_unflatten(fake_args, mod._in_spec)
-        fake_mode = _detect_fake_mode_from_gm(mod)
+        # TODO T204030333
```

Comment:
Is it because of tracing with fake tensors or something else?

Reply:
I didn't investigate further. Possibly..